internal/boxcli/run.go

```diff
@@ -53,6 +54,18 @@ func listScripts(cmd *cobra.Command, flags runCmdFlags) []string {
 }
 
 func runScriptCmd(cmd *cobra.Command, args []string, flags runCmdFlags) error {
+	if len(args) == 0 {
+		scripts := listScripts(cmd, flags)
+		if len(scripts) == 0 {
+			return redact.Errorf("no scripts defined in devbox.json")
+		}
+		fmt.Println("Available scripts:")
+		for _, p := range scripts {
+			fmt.Printf("* %s\n", p)
```

Comment:
Could we change these to explicitly take in the stdout writer, like `fmt.Fprintln(cmd.OutOrStdout(), "<msg>")`? 

That way, it'll make it easier to change the writer for tests or the --quiet flag, or similar future use-cases.

Reply:
ah, you beat me to it :)